<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_stackedProfilBoxes</class>
 <widget class="QDialog" name="Dialog_stackedProfilBoxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>314</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>314</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>431</width>
    <height>314</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Box edition</string>
  </property>
  <widget class="QPushButton" name="pushButton_addBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Add boxes</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Widget_stackedprofilbox_edit" name="widget_stackedProfBox_edit" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>301</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_aboutBoxesList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_empty_addingBox">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 220);
border-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_with_boxesList">
    <widget class="QTableView" name="tableView_boxesList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>221</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>180</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_removeAllBoxes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>278</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Remove all</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_stackedprofilbox_edit</class>
   <extends>QWidget</extends>
   <header>widget_stackedprofilbox_edit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView_boxesList</tabstop>
  <tabstop>pushButton_addBox</tabstop>
  <tabstop>pushButton_removeAllBoxes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
